Sources/StreamVideo/Utils/Sorting.swift

```diff
@@ -34,13 +34,13 @@ public let defaultComparators: [StreamSortComparator<CallParticipant>] = [
 /// - `ifInvisible(publishingAudio)`: Sorts participants based on their audio status, but only if they are not visible.
 /// - `roles()`: Sorts participants based on their assigned roles.
 public let livestreamComparators: [StreamSortComparator<CallParticipant>] = [
-    ifInvisible(dominantSpeaker),
-    ifInvisible(isSpeaking),
-    ifInvisible(publishingVideo),
-    ifInvisible(publishingAudio),
-    roles(),
-    joinedAt,
-    userId
+    combineComparators([
```

Comment:
why do we need to change this?

Reply:
This one is changed to match React sorting comparators as they seem to have been updated https://github.com/GetStream/stream-video-js/blob/3d115dd349fb0f4c57ff162dbaf829c2f96b45b9/packages/client/src/sorting/presets.ts#L93